ui/component/header/view.jsx

```diff
@@ -324,26 +325,26 @@ const Header = (props: Props) => {
                       )}
                     </MenuButton>
                     <MenuList className="menu__list--header">
-                      <MenuItem className="menu__link" onSelect={() => history.push(`/$/${PAGES.UPLOADS}`)}>
```

Comment:
Come on, don't make function for a mere link. I'd assume, React does all that in a batch anyway...